### PR TITLE
igushkin/gradle-6-3

### PR DESCRIPTION
**Pull request by h0tk3y**

### CODE REVIEW

**Comment by turansky** [comment]
@h0tk3y Why gradle wrapper not updated fully? 
Automatic update [generate more changes](https://github.com/Kotlin/kotlinx.coroutines/pull/1971/commits/ec446e6e85eb656bfa23c66e0a8be5213eb21500), (not only in properties).

File: integration-testing/build.gradle [hunk 1]
```diff
@@ -56,7 +56,6 @@ task npmTest(type: Test) {
 task mavenTest(type: Test) {
     def sourceSet = sourceSets.mavenTest
     dependsOn(project(':').getTasksByName("publishToMavenLocal", true))
-    dependsOn.remove(project(':').getTasksByName("dokka", true))
```
Comment (qwwdfsad) [inline integration-testing/build.gradle:59]: Please elaborate on why did you remove this line

Comment (turansky) [reply integration-testing/build.gradle:59]: No `remove` in 6.3 :(

Comment (elizarov) [reply integration-testing/build.gradle:59]: I do belive we still build all docs no CI now because the docs are needed for `knitCheck` task which we run in CI to ensure that we catch fast the cases where somebody committed updates to docs and forgot to run `knit` task.
